### PR TITLE
Feature/357 update thr firing remainder module

### PR DESCRIPTION
**Pull request by JulianHammerl**
* **Tickets addressed:** Confluence 357
* **Review:** By commit  
* **Merge strategy:** Merge (no squash)  

## Description

The logic of the first module call was changed. Now, the only difference between the first module call and all subsequent calls is the computation of the control period (fsw rate). The remaining module code is now always executed (also for first call), so any center of mass offset information coming from the THRArrayCmdForceMsg is now always considered.

## Verification

The UnitTest was updated accordingly.

## Documentation

The documentation was updated accordingly.

## Future work

None.

### CODE REVIEW

**Review by thibaudteil — APPROVED** [review]
Looks good

**Comment by codyallard28** [comment]
@JulianHammerl, should the commit that defaulted the timestep to 2. even show up in the history? I am not an expert on interactive rebasing or atomic commits... so this is purely a question.

**Comment by JulianHammerl** [comment]
> @JulianHammerl, should the commit that defaulted the timestep to 2. even show up in the history? I am not an expert on interactive rebasing or atomic commits... so this is purely a question.

@codyallard28 I decided against squashing these commits together because the purpose of the first commit is to include the module logic into the first time step, and the purpose of the second commit is to make the time step of 2 seconds changeable. I am happy to squash them together though if desired @patkenneally

File: src/fswAlgorithms/effectorInterfaces/thrFiringRemainder/_UnitTest/test_thrFiringRemainder.py
Comment (patkenneally) [inline src/fswAlgorithms/effectorInterfaces/thrFiringRemainder/_UnitTest/test_thrFiringRemainder.py]: :small_red_triangle_down: Delete this. The Git history contains all this information

**Review by patkenneally — APPROVED** [review]
Legend:
:small_red_triangle_down: *Issues* to address before merge
:large_orange_diamond: *Requests* that should not block merge, but should at least be discussed
:large_blue_circle: *Recommendations* that can be ignored if desired

File: src/fswAlgorithms/effectorInterfaces/thrFiringRemainder/_UnitTest/test_thrFiringRemainder.py
Comment (JulianHammerl) [reply src/fswAlgorithms/effectorInterfaces/thrFiringRemainder/_UnitTest/test_thrFiringRemainder.py]: Done.

**Pull request merged**
